Makefile

```
@@ -75,6 +75,8 @@ bin/%/submariner-route-agent: $(shell find pkg/routeagent_driver)
 bin/%/submariner-globalnet: $(shell find pkg/globalnet)
 	GOARCH=$(call dockertogoarch,$(patsubst bin/linux/%/,%,$(dir $@))) ${SCRIPTS_DIR}/compile.sh $@ ./pkg/globalnet
 
+bin/%/await-node-ready: $(shell find pkg/await_node_ready)
```

This also uses `pkg/node` and `pkg/versions`.

```suggestion
bin/%/await-node-ready: $(shell find pkg/await_node_ready pkg/node pkg/versions)
```

---

Ok but is that needed? Route agent only specifies `$(shell find pkg/routeagent_driver)` but it uses other packages as well. Same with globalnet.

---

It doesn’t affect the binary that gets built, it only determines when it gets rebuilt. But this approach is inherently brittle, I’ve got a better one — let’s merge this and I’ll fix all the binaries properly.